ruleset/rules/0575-win-base_rules.xml

```diff
@@ -1,14 +1,12 @@
 <!--
-  -  Windows Event Channel ruleset
-  -  Created by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
-<!-- Group of parent rules for the Windows eventchannel -->
+<!--
+  Powershell Rules 60000 - 60099
```

Comment:
This is not correct, as these rules group different windows events. perhaps Windows Event Channel rules is better?

Reply:
```suggestion
  Eindows eventchannel rules 60000 - 60099
```